indico/modules/categories/templates/management/content.html

```diff
@@ -74,8 +76,15 @@
 {% endblock %}
 
 {% block content %}
-    {{ subcategories_table(subcategories) }}
-    <script>
-        setupCategoryTable();
-    </script>
+    {% if subcategories %}
+        {{ subcategories_table(subcategories) }}
+        <script>
+            setupCategoryTable();
+        </script>
+    {% elif current_category.events %}
```

Comment:
We want to support both in case there's both in the same category (there are some legacy category which actually have this right now for whatever strange reason)